<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gameScene</class>
 <widget class="QMainWindow" name="gameScene">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>40</y>
      <width>121</width>
      <height>321</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="startButton">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>45</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>45</height>
        </size>
       </property>
       <property name="text">
        <string>开始游戏</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>10</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="stopButton">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>45</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>45</height>
        </size>
       </property>
       <property name="text">
        <string>暂停游戏</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>10</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="tipButton">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>45</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>45</height>
        </size>
       </property>
       <property name="text">
        <string>提示</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>10</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="resetButton">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>45</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>45</height>
        </size>
       </property>
       <property name="text">
        <string>重排</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_4">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>10</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="backButton">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>45</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>45</height>
        </size>
       </property>
       <property name="text">
        <string>返回主界面</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>480</y>
      <width>98</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="settingButton">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>45</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>45</height>
        </size>
       </property>
       <property name="text">
        <string>设置</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="helpButton">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>45</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>45</height>
        </size>
       </property>
       <property name="text">
        <string>帮助</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="timeLabel">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>490</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>19</pointsize>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QProgressBar" name="timeprogressBar">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>530</y>
      <width>591</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
    <property name="format">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="gameArea" native="true">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>640</width>
      <height>400</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>640</width>
      <height>400</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>640</width>
      <height>400</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
